### DISCUSSION TITLE
Unable to display api(downloaded) image url on Image component (Android tv)

### DISCUSSION THREAD

**Discussion by avishka-codimite** [react-native-tvos/react-native-tvos#387] [Q&A] [answered]
I'm struggling to display images on my application (Android tv). I'm getting this kind of image URL to show pictures from the API side. Seems to be image URL does not work with the Image component. Also, I tried on a react native android mobile application. It's worked fine. Hard to figure out why not support Android tv. 

URL = https://d3k2f0s3vqqs9o.cloudfront.net/media/911C7982-90BA-4BE7-AC78061C6F4C0B41/A6C58A1A-ACE2-4164-9C2ECCB104F66640/C5075FFD-9029-4419-A2FCA11EF8E70176/webimage-D8E15E0E-045A-4B5C-BF42AEA4A752AAC7.png

` <Image style={{width: 100, height: 100}} source={{uri: 'https://d3k2f0s3vqqs9o.cloudfront.net/media/911C7982-90BA-4BE7-AC78061C6F4C0B41/A6C58A1A-ACE2-4164-9C2ECCB104F66640/C5075FFD-9029-4419-A2FCA11EF8E70176/webimage-D8E15E0E-045A-4B5C-BF42AEA4A752AAC7.png'}} resizeMode="cover" /> `

**Comment by douglowder** [comment, marked as answer]
I created a new RN project using the latest react-native-tvos template (0.69.6-0), and added your Image markup above to the default app from the template. It seems to work fine on Android TV.
<img width="1368" alt="Screen Shot 2022-11-10 at 11 48 09 PM" src="https://user-images.githubusercontent.com/6577821/201291907-daf3d69c-733c-476d-80a4-38547523b453.png">
